source/overview/license-and-subscription.rst

```diff
@@ -71,13 +71,33 @@ You may incur retroactive charges for any unique users added that exceed the lic
 Renewing an Enterprise license
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-The ability to renew your license in the Customer Portal will be available in a future release.
+You can renew your Mattermost license with a credit card if you have a standard Mattermost contract. When you renew your license, you can also increase the number of active users.
 
-To renew your license in the meantime, contact your Customer Success Manager to start the renewal process. Alternatively, `contact us <https://mattermost.com/contact-us/>`_ for assistance.
+If you are a reseller, have a non-standard contract, or want to adjust the number of active users on your license please contact sales@mattermost.com.
+
+System Admins will be alerted 60 days prior to the license expiry via a banner in Mattermost. Select **Renew license now** to start the renewal process in the Customer Portal. You can also dismiss the banner and renew your license at a later date via **System Console > Edition and License**.
```

Comment:
We mention in the FAQ that the banner returns on browser refresh - @jasonblais, is this still accurate? And what about non-browser based users?

Reply:
This should still be true, but you can confirm with Steve Mudie (QA). It should work the same for desktop app users, but no such banner is shown on the mobile apps.

Reply:
@stevemudie, does the banner still behave as mentioned in the docs - i.e., can be closed but then reappears after refresh?

Reply:
We should document (or otherwise clarify) that this new flow is only applicable to v5.32 and later. Perhaps we can say that for those servers that have not upgraded to v5.32 can contact support to renew their license.

Reply:
> We mention in the FAQ that the banner returns on browser refresh - @jasonblais, is this still accurate? And what about non-browser based users?

This is correct, the banner is dismissible but will come back when they refresh (or reload in the desktop app) within the 60 days before it expires. However, once expired, the banner is no longer dismissible and after the 10 days after expiration, enterprise features are disabled and the banner is changed to: 

<img width="1240" alt="Screen Shot 2021-01-26 at 7 32 56 PM" src="https://user-images.githubusercontent.com/936315/105925002-5e6d6100-600d-11eb-9826-5ed2cad3f1a8.png">

Reply:
> We have [FAQs around that](https://docs.mattermost.com/overview/license-and-subscription.html#what-happens-to-my-license-if-i-don-t-renew-in-time) which also list the features that are affected. LMK if we need more. :)

Perfect, didn't know that was there.